code/lib/cli/src/automigrate/fixes/remove-jest-testing-library.test.ts

```diff
@@ -0,0 +1,55 @@
+import { expect, it } from 'vitest';
+
+import type { StorybookConfig } from '@storybook/types';
+import type { JsPackageManager } from '@storybook/core-common';
+import { removeJestTestingLibrary } from './remove-jest-testing-library';
+
+const check = async ({
+  packageManager,
+  main: mainConfig = {},
+  storybookVersion = '8.0.0',
+}: {
+  packageManager: Partial<JsPackageManager>;
+  main?: Partial<StorybookConfig> & Record<string, unknown>;
+  storybookVersion?: string;
+}) => {
+  return removeJestTestingLibrary.check({
+    packageManager: packageManager as any,
+    configDir: '',
+    mainConfig: mainConfig as any,
+    storybookVersion,
+  });
+};
+
+it('should show incompatible addons registered in main.js', async () => {
```

Comment:
Is it really about the addons which are registerd in main.js. It's rather about installed dependencies, isn't it. So the title of this test doesn't quite fit.

Reply:
fixed